src/components/enter-mfa/index.njk

```diff
@@ -3,15 +3,26 @@
 {% from "govuk/components/button/macro.njk" import govukButton %}
 {% from "govuk/components/details/macro.njk" import govukDetails %}
 {% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
 {% set pageTitleName = 'pages.enterMfa.title' | translate %}
+{% set lastThreeDigits = phoneNumber.slice(-3) %}
 
 {% block content %}
 
   {% include "common/errors/errorSummary.njk" %}
 
   <h1 class="govuk-heading-l govuk-!-margin-top-0 govuk-!-margin-bottom-3">{{'pages.enterMfa.header' | translate }}</h1>
 
-  <p class="govuk-body">{{'pages.enterMfa.info.paragraph1' | translate }}</p>
+  {% set lastThreeDigits = phoneNumber.slice(-3) %}
+
+  {% set insetText %}
+    {{ 'pages.checkYourPhone.info.paragraph1' | translate }} <span class='govuk-!-font-weight-bold'>{{ lastThreeDigits }}</span>
```

Comment:
Is `pages.enterMfa.info.paragraph1` being used anywhere else? Otherwise I think we should keep using `pages.enterMfa.info.paragraph1` and update the text in the translation files